project-1/gspread_code.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Oct 16 15:12:29 2023
+
+@author: computer world
+
+Project : getting data from github repositories(through selenium library) and 
+sending to google spreadsheet with gspread package
+"""
+
+import gspread
+import pandas as pd
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+path = "documents/datasets_for_data_science/gspread_project/spartan-figure-402211-2241d0349aab.json"
```

Comment:
I'd advice you use relative path (from your current working directory) instead of absolute path. This will avoid errors when running this code on another computer. Check the link below for more info on relative and absolute paths.

https://courses.cs.washington.edu/courses/cse140/13wi/file-interaction.html#:~:text=Absolute%20and%20Relative%20file%20paths&text=An%20absolute%20file%20path%20describes,For%20example%2C%20example_directory.